auth.go

```diff
@@ -520,6 +575,30 @@ func (x *Central) authenticate(identity, password string) (AuthUser, error) {
 	}
 }
 
+func (x *Central) CheckAdmin(userId UserId) (bool, error) {
```

Comment:
`Check` is a classic no-no for function names. Rather call it `IsAdmin`.

I would split this into two functions. One of them should just be generic, and search for the user's presence in any group (ie pass the magic string "admin" in as a function parameter).

Also, put a clear comment here, saying that the word "admin" is the magic group name used by the imqsauth. This is hack, bringing in knowledge from the imqsauth package, but we feel it's OK for this particular edge case.